### DISCUSSION TITLE
Two Stores called in a Parent and Child Component conflict

### DISCUSSION THREAD

**Discussion by joaopedrosdmm** [pmndrs/zustand#1672] [Q&A]
## Summary
I have created two stores, called store.js and websiteStore.js with two hooks, respectively useStore and useWebsite. When I call useWebsite in a Parent Component, I cannot use useStore in the Child Component, it simply doesn't work. If I remove the useWebsite and replace it with useStore, it starts working.
Sorry for the mediocre explanation, but that is it.

## Link to reproduction

Don't have

## Check List

Please do not ask questions in issues.

- [ ] I've already opened a [discussion](https://github.com/pmndrs/zustand/discussions) before opening this issue, or already discussed in other media.

Please include a minimal reproduction.

- [ ] I've added a link to a [typescript playground](https://www.typescriptlang.org/play) or [codesandbox](https://codesandbox.io) with a minimal reproduction.

Please check this if you're filing an issue regarding TypeScript.

- [ ] I've read the [typescript guide](https://github.com/pmndrs/zustand/blob/main/docs/guides/typescript.md), in particular that `create` is to be used as `create<T>()(...)` and not `create<T>(...)`.

**Comment by dai-shi** [comment]
If you can't provide a minimal reproduction, can you create a small code example to illustrate your problem?
